pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -58,7 +58,10 @@ var (
 	ResourceAWSPodENI v1.ResourceName = "vpc.amazonaws.com/pod-eni"
 
 	LabelInstanceHypervisor      = LabelDomain + "/instance-hypervisor"
+	LabelInstanceCategory        = LabelDomain + "/instance-category"
 	LabelInstanceFamily          = LabelDomain + "/instance-family"
```

Comment:
Instead of adding category as a new label, we could add this field as a second value of `instance-family`, which would allow us to keep compatibility while moving new customers over to this new scheme.